README.md

```
@@ -1,337 +1,288 @@
+- **Rick and Morty REST API**: Fonte de dados oficial
+
+### **Arquitetura**
+- **StatefulWidget**: Gerenciamento de estado local
+- **Repository Pattern**: Abstração da camada de dados
+- **Clean Architecture**: Separação de responsabilidades
+
+---
+
+## 🚀 Como Executar
+
+### **Pré-requisitos**
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)